libraries/Servo/src/Servo.h

```diff
@@ -46,10 +46,10 @@
 
 // the following are in us (microseconds)
 //
-#define MIN_PULSE_WIDTH       544     // the shortest pulse sent to a servo  
-#define MAX_PULSE_WIDTH      2400     // the longest pulse sent to a servo 
-#define DEFAULT_PULSE_WIDTH  1500     // default pulse width when servo is attached
-#define REFRESH_INTERVAL    20000     // minumim time to refresh servos in microseconds 
+#define MIN_PULSE_WIDTH       800     // the shortest duty cycle sent to a servo  
```

Comment:
what are the values from the standard Arduino servo library?

Reply:
The ones in master are the same as in Arduino's Servo library.  This PS restricts the values to a smaller range.

````
#define MIN_PULSE_WIDTH       544     // the shortest pulse sent to a servo  
#define MAX_PULSE_WIDTH      2400     // the longest pulse sent to a servo 
#define DEFAULT_PULSE_WIDTH  1500     // default pulse width when servo is attached
#define REFRESH_INTERVAL    20000     // minumim time to refresh servos in microseconds 
````

Reply:
@earlephilhower No, no restrictions now in the updated PR, but safer defaults, it is no one's best interest to destroy quality manufacturer (Tower Pro) servos for people that just unsuspectingly try the example for the first time.